pkgs/tools/nix/nixos-option/nixos-option.sh

```diff
@@ -0,0 +1,51 @@
+#! @runtimeShell@
```

Comment:
`shellcheck` is a nice tool to enable.

Reply:
Good point, i haven't really used it since I haven't written many "real" (not just for my own use) shellscripts. I didn't see any specific hooks for it but I guess I can just add `# shellcheck shell=bash` and explicitly invoke `shellcheck` on the script in the checkPhase